libcxx/docs/Status/Cxx23Issues.csv

```diff
@@ -46,7 +46,7 @@
 "`3473 <https://wg21.link/LWG3473>`__","Normative encouragement in non-normative note","November 2020","|Complete|","15.0","|format|"
 "`3474 <https://wg21.link/LWG3474>`__","Nesting ``join_views`` is broken because of CTAD","November 2020","|Complete|","15.0","|ranges|"
 "`3476 <https://wg21.link/LWG3476>`__","``thread`` and ``jthread`` constructors require that the parameters be move-constructible but never move construct the parameters","November 2020","",""
-"`3477 <https://wg21.link/LWG3477>`__","Simplify constraints for ``semiregular-box``","November 2020","","","|ranges|"
+"`3477 <https://wg21.link/LWG3477>`__","Simplify constraints for ``semiregular-box``","November 2020","|Complete|","13.0","|ranges|"
```

Comment:
Are you sure about the version number? LLVM 13 was released on the 4th Oct 2021 and the review is from the 28th of May 2023. Based on that information I would expect LLVM 17, which matches with the tags listed in https://github.com/llvm/llvm-project/commit/9f6439f1c5a3e98d03b7416449a256062e420003

Reply:
Sorry, I've forgot to update the summary. Since https://reviews.llvm.org/D102135 (6829db727e9e67dfdb70dd0846ffd4e48e00a98d), we have addressed this issue, The version of LLVM at that time was 13.0 https://github.com/llvm/llvm-project/blob/6829db727e9e67dfdb70dd0846ffd4e48e00a98d/libcxx/docs/ReleaseNotes.rst

Reply:
Then 13.0 indeed seems the right version.